packages/trader/src/Modules/Trading/Components/Form/screen-small.tsx

```diff
@@ -242,7 +248,20 @@ const ScreenSmall = observer(({ is_trade_enabled }: { is_trade_enabled: boolean
             <TradeParamsLoader speed={2} />
         </div>
     ) : (
-        <CollapsibleTradeParams has_allow_equals={has_allow_equals} {...collapsible_trade_params_props} />
+        <>
```

Comment:
is the fragment needed? :)

Reply:
Seems so, because there is a ternary inside

Reply:
yes just because If I remove the fragment I get eslint warning of not using multiple ternary operator . With Fragment , it treats it differently